Application/Items/Commands/CreateItemCommand.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Application.Common.Exceptions;
+using Application.Common.Interfaces;
+using Domain.Entities;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Items;
+
+public record CreateItemCommand : IAuthorizedRequest<ItemDto>
+{
+    public Employee Employee;
+    public string? Title { get; init; }
+    public string? Category { get; init; }
+    public decimal Price { get; init; }
+    public string? Description { get; init; }
+    public string? Brand { get; init; }
+    public string? Photo { get; init; }
+
+    public async Task<bool> Authorize(Employee employee, IUserService userService, IApplicationDbContext dbContext)
+    {
+        this.Employee = employee;
+        return await userService.CanCreateItemAsync(employee);
```

Comment:
CanManageTenant()

Reply:
wait why? How does it relate to managing tenants?

Reply:
perhaps the naming is not the best, but CanViewTenant is for regular employees (f.e. cashiers), whereas CanManageTenant is for managers and admins. a manager can edit, create, delete all items, so I don't see where its helpful to have another function which does the same thing?

Reply:
`CanViewTenant` expects `tenantId`, but when creating a new item, there's no `tenantId` to check, so it wouldn't work

Reply:
So I guess I should still create `CanCreateTenant` method which ckecks that employee has position manager or admin?

Reply:
rewrote Create Item endpoint to take `tenantId` from path